src/js/client.ts

```diff
@@ -130,8 +130,8 @@ export class ReactNativeClient extends BaseClient<ReactNativeClientOptions> {
    * @inheritDoc
    */
   public init(): void {
-    super.init();
     this._initNativeSdk();
+    super.init();
```

Comment:
Q: Was it previously wrong and you fix it now or is it a change that was required for V8?

Reply:
The `init` was added for V8, the native init needs to happen before the JS init, to avoid the native not available error log in Expo.

Before v8 the native init happened in constructor, which doesn't make sense with dedicate init method.